### ISSUE TITLE
Primefaces CKeditor bug url

### ISSUE THREAD

**Issue by Kalmaasaly** [primefaces-extensions/primefaces-extensions.github.com#740]
Could any one knows how to solve this bug resources-ckeditor generate this link which is wrong: /javax.faces.resource/ckeditor/plugins/mathjax/?ln=primefaces-extensions&v=7.0.2images/loader.gif

This url works correctly /javax.faces.resource/ckeditor/plugins/mathjax/images/loader.gif?ln=primefaces-extensions&v=7.0.2

I am using primefaces 7 primefaces-extensions 7.0.2 resources-ckEditor 7.0.2

**Comment by melloware** [comment]
I can't reproduce your issue.  Can you create a small reproducible sample using the Test project please?  Clone this repository https://github.com/primefaces/primefaces-test.git in order to reproduce your problem, you'll have better chance to receive an answer and a solution.

I can only debug it if I have an example of the problem happening.

**Comment by melloware** [comment]
I think what you really want is an enhancement to use the MathJax plugin with CKEDitor?

**Comment by Kalmaasaly** [comment]
Yes, I want to use MathJax plugin but Error URL found also in mono skin doesn't  render the icons of ckEditor

**Comment by melloware** [comment]
Added MathJAX plugin in 8.0.4 see commit: https://github.com/primefaces-extensions/resources-ckeditor/commit/e9f7cc8673a87300ec8be9904dca7044c1480054

**Comment by melloware** [comment]
Oh and I also fixed your images/loader.gif issue reported in the original issue.

**Comment by Kalmaasaly** [comment]
Good Job
bye the way I implemented MathJax in resources-ckEditor old version
